securedrop/Makefile

```diff
@@ -1,6 +1,16 @@
 TAG ?= $(shell git rev-parse HEAD)
 IMAGE = securedrop
 
+.PHONY: lint
+lint:
+	pylint --reports=no --errors-only \
+		--disable=import-error \
+		./*.py ./**/*.py
```

Comment:
<pre>
./**/*.py
</pre>

is the same as


<pre>
./*/*.py
</pre>

and it won't catch the *.py that are nested at the third level.